README.md

```diff
@@ -33,26 +33,17 @@ const settings = {
       {
         code: "USD", // Set currency code. *
         symbol: "$", // Set currency symbol. *
-        rates: { // Set currency rate to all currencies in list.
-          "€": 1.1, // EUR to USD
-          "₽": 0.011, // RUB to USD
-        },
+        rateToBase: 1, // Set the ratio of the base currency to the currency from the list.
       },
       {
         code: "EUR",
         symbol: "€",
-        rates: {
-          "$": 0.9, // USD to EUR
-          "₽": 0.01, // RUB to EUR
-        },
+        rateToBase: 0.9,
```

Comment:
Euro usually bigger than dollar, isn't it?

Reply:
![image](https://github.com/molotow11/JsSimpleCart/assets/98450298/c5f1f40b-d8c6-4716-b5db-fcc05ab52761)
Yes, but now the base currency is set to USD)

Reply:
As you can see 1 dollar is 0.9 euro.

Reply:
![image](https://github.com/molotow11/JsSimpleCart/assets/98450298/a90ba242-8ef2-4c44-9bc8-d64bda7c7e5d)
If the base currency is the euro, then you need to set a different rate

Reply:
Please, read it properly:
1) 1 United states Dollar equals 0.91 Euro
2) 1 Euro equals 1.10 USD
It is correct, but in your fixes it is not correct. Because base currency is USD, so Euro should be 1.10 then.

Reply:
Yes, it really turned out not logical, thanks for noticing!